src/ui/TUI.java

```diff
@@ -2,48 +2,223 @@
 
 import dto.User;
 
+import java.util.ArrayList;
+
+import java.util.Scanner;
+
 /**
- * Created by freya on 14-02-2017.
+ * Wrote by Kasper on 14-02-2017.
  */
 public class TUI implements UI
 {
-    @Override
-    public void printWelcomeMsg() {
+    private Scanner scanner = new Scanner(System.in);
 
+    @Override
+    public void printMainMenu()
+    {
+        System.out.println("Choose a following option by typing the number \n" +
+                "1 - Create User \n" +
+                "2 - Show Users \n" +
+                "3 - Edit User \n" +
+                "4 - Delete User \n" +
+                "5 - Exit program");
     }
 
     @Override
-    public void printMsg(String message) {
+    public void printMsg(String message)
+    {
+        System.out.println(message);
+    }
 
+    @Override
+    public void listUsers(ArrayList<User> listOfUser)
+    {
+        for (User user : listOfUser)
+        {
+            System.out.println(user.toString());
```

Comment:
Fint for nu, men jeg tænker at vi med siden gør det lidt tydeligere hvad der sker, men lidt mere beskrivende tekst.